docs/configuration.asciidoc

```diff
@@ -764,3 +764,11 @@ The unit is provided as suffix directly after the number, without and separation
  * `gb` (gigabytes)
 
 NOTE: we use the power-of-two sizing convention, e.g. `1 kilobyte == 1024 bytes`
+
+[float]
```

Comment:
Can you move this to right before "Generic Environment variables", and then add a table similar to the other "non-generic" config options?

(My previous comment of "append it at the end" didn't consider that we have a bunch of "special" stuff at the end, like the generic environment variables and the configuration format docs, sorry about that)

Reply:
Done.